app/v1/controllers/AmendPensionsReliefsController.scala

```diff
@@ -110,7 +110,7 @@ class AmendPensionsReliefsController @Inject() (val authService: EnrolmentsAuthS
           RuleTaxYearNotSupportedError | MtdErrorWithCustomMessage(ValueFormatError.code) =>
         BadRequest(Json.toJson(errorWrapper))
       case InternalError => InternalServerError(Json.toJson(errorWrapper))
-      case _               => unhandledError(errorWrapper)
+      case _             => unhandledError(errorWrapper)
```

Comment:
The format of this 'errorResult' method needs to be in the same format of the CreateAmendUkDividendsAnnualSummaryController.errorResult. 

With the 'MtdErrorWithCustomMessage(ValueFormatError.code)', can we not just write this as 'ValueFormatError'?

Reply:
made the changes to the errorResult method @harry-ritchie